### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by bhargaviRdy**
Made the landing page with minimal design and stuff. The content will be modified later.
Live Site:  https://vibeui-bhargavi.netlify.app/

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    justify-content: center;
+    gap: 4rem;
+    flex-wrap: wrap;
+}
+.content{
+    width: 400px;
+    font-size: 1rem;
+    line-height: 1.5;
+}
+p{
```
Comment (tusharkandpal) [inline styles.css]: avoid using p tag directly

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+iframe{
+    width: 100%;
+    border: none;
+}
+
+.footer{
+    position: absolute;
+    left: 0;
+    right: 0;
+    /* bottom: 0; */
```
Comment (tusharkandpal) [inline styles.css]: remove this comment please

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
suggested few minor changes & would suggest you to format your code a little bit.. else is good👍🏻

File: styles.css (hunk 2 above)
Comment (bhargaviRdy) [reply styles.css]: Sure, Definitely

File: styles.css (hunk 1 above)
Comment (bhargaviRdy) [reply styles.css]: yea, I will refactor this code once, I'm done with all the components. I will follow the best practices. Thanks

File: index.html [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+                </p>
+                <div class="btns">
+                    <a class="btn btn-primary" href="./docs.html">Documentation</a>
+                    <a class="btn btn-outline" href="#installation">Installation</a>
+                </div>
+
+            </div>
+            <div class="image">
+                <img src="images/website-builder.svg" alt="ghghg">
+
```
Comment (harshitadaswani) [inline index.html]: remove empty newline here

File: index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                    adipisicing elit. Quo, ea?
+                </p>
+                <div class="btns">
+                    <a class="btn btn-primary" href="./docs.html">Documentation</a>
+                    <a class="btn btn-outline" href="#installation">Installation</a>
+                </div>
+
+            </div>
+            <div class="image">
+                <img src="images/website-builder.svg" alt="ghghg">
```
Comment (harshitadaswani) [inline index.html]: give better alt names

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    display: inline;
+    margin: 1rem;
+}
+.image{
+    width: 400px;
+}
+
+.image img{
+    width: 300px;
+}
```
Comment (harshitadaswani) [inline styles.css]: use rem or % units for responsiveness

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+.row{
+    display: flex;
+    padding: 3rem 3rem 0 3rem;
+    align-items: center;
+    justify-content: center;
+    gap: 4rem;
+    flex-wrap: wrap;
+}
+.content{
+    width: 400px;
```
Comment (harshitadaswani) [inline styles.css]: use rem or % units

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+    font-size: 1rem;
+    line-height: 1.5;
+}
+p{
+    margin-top: 20px;
```
Comment (harshitadaswani) [inline styles.css]: same here

File: styles.css [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+}
+.btns .btn:not(:last-child){
+    margin-right: 0.625rem;
+
+}
+.code-display{
+    width: 400px;
```
Comment (harshitadaswani) [inline styles.css]: same here

File: styles.css [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+    border: none;
+}
+
+.footer{
+    position: absolute;
+    left: 0;
+    right: 0;
+    /* bottom: 0; */
+    margin-top: 2rem;
+    background: whitesmoke;
```
Comment (harshitadaswani) [inline styles.css]: define color in a variable and then use it

**Review by harshitadaswani — CHANGES_REQUESTED** [review]
Be consistent about the units you are using.

**Review by harshitadaswani — APPROVED** [review]
Yes, it looks good! You can merge it.

**Review by tusharkandpal — APPROVED** [review]
Code looks good ! Go ahead.

**Pull request merged**